src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -306,18 +323,31 @@ Task AbandonAsyncOnCancellation(BrokeredMessage message)
             {
                 logger.InfoFormat("Abandoning brokered message {0}", message.MessageId);
 
-                await message.SafeAbandonAsync(propertiesToModify).ConfigureAwait(false);
+                if (await message.SafeAbandonAsync(propertiesToModify).ConfigureAwait(false))
+                {
+                    logger.InfoFormat("Brokered message {0} abandoned successfully.", message.MessageId);
+                }
+                else
+                {
+                    logger.InfoFormat("Abandoning brokered message {0} failed. Message will reappear after peek lock duration is over.", message.MessageId);
```

Comment:
I'm good with a WARN. Not sure it is required though since we are in PeekLock here and the message would reappear anyway.